service/src/main/java/bio/terra/profile/service/gcp/GcpService.java

```diff
@@ -0,0 +1,63 @@
+package bio.terra.profile.service.gcp;
+
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.profile.service.crl.GcpCrlService;
+import bio.terra.profile.service.gcp.exception.InaccessibleBillingAccountException;
+import bio.terra.profile.service.profile.exception.MissingRequiredFieldsException;
+import com.google.cloud.billing.v1.BillingAccountName;
+import com.google.iam.v1.TestIamPermissionsRequest;
+import com.google.iam.v1.TestIamPermissionsResponse;
+import java.util.List;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class GcpService {
+
+  private static final Logger logger =
+      LoggerFactory.getLogger(bio.terra.profile.service.gcp.GcpService.class);
+
+  public static final List<String> BILLING_ACCOUNT_PERMISSIONS_TO_TEST =
+      List.of("billing.resourceAssociations.create");
```

Comment:
If it's always just one value maybe can use Collections.singletonList() instead.

Reply:
👍 I'll check that out. This was pulled out of the old step, but happy to make that change